packages/fxa-settings/public/locales/en-US/settings.ftl

```diff
@@ -0,0 +1,12 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+## branding
+project-brand = Firefox Accounts
+-brand-name-mozilla = Mozilla
+-brand-name-firefox = Firefox
+
+## Display name page
```

Comment:
@chenba While building out parts of this I noticed that we may potentially have a bunch of smaller ftl files. If  flow reuses components (input, password, checkbox etc), would the ftl files live in those component folders.

Reply:
The plan is to have ftl files next to the components where they are used.  This file (settings.ftl) will be a merged file of all those small ftl files, and not to be manually edited.  But it is the file that is served to the client (and copied to the l01n repo for the localizers), as you are doing in the PR.